:'(

Its now finally working on macOs for me, so that is great!

But... 
Windows fails for me...
Ive been searching for the cause but am unsure, maybe it helps to compare conan settings?

My conan profile is:
```
[settings]
arch=x86_64
build_type=Release
compiler=msvc
compiler.cppstd=14
compiler.runtime=dynamic
compiler.version=193
os=Windows
[conf]
tools.microsoft.msbuild:vs_version=17
tools.cmake.cmaketoolchain:generator=Ninja
[tool_requires]
```

Relevant line (maybe):
```
/bin/sh ../libtool --mode=compile --tag=RC C:/Program Files (x86)/Windows Kits/10/bin/10.0.22000.0/x64/rc.exe `/bin/sh /c/users/joris/.conan2/p/b/libic587608ee0597f/b/src/lib/../windows/windres-options --escape 1.17` -i /c/users/joris/.conan2/p/b/libic587608ee0597f/b/src/lib/../windows/libiconv.rc -o libiconv.res.lo --output-format=coff
/bin/sh: -c: line 1: syntax error near unexpected token `('
/bin/sh: -c: line 1: `/bin/sh ../libtool --mode=compile --tag=RC C:/Program Files (x86)/Windows Kits/10/bin/10.0.22000.0/x64/rc.exe `/bin/sh /c/users/joris/.conan2/p/b/libic587608ee0597f/b/src/lib/../windows/windres-options --escape 1.17` -i /c/users/joris/.conan2/p/b/libic587608ee0597f/b/src/lib/../windows/libiconv.rc -o libiconv.res.lo --output-format=coff'
make[1]: *** [Makefile:99: libiconv.res.lo] Error 2
```

You have windows kit installed in `Program Files (x86)` too?

--- a/conanfile.txt
+++ b/conanfile.txt
@@ -8,7 +8,6 @@
 
 [requires]
 boost/[>=1.82.0 <2.0.0]
-libiconv/1.17
 zlib/1.3
 libarchive/3.6.2
 zstd/1.5.2

